cassandra-dac/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java

```diff
@@ -449,6 +449,9 @@ private Response getRecordByIdentifier(
                   CassandraUtil.createQuery(x.getKey(), x.getValue(), selectWhere);
                 });
       }
+      ProjectLogger.log(
+          "CassandraOperationImpl:getRecordByIdentifier - Query : " + selectWhere.getQueryString(),
```

Comment:
Change ...

From:
"CassandraOperationImpl:getRecordByIdentifier - Query : "

To:
"CassandraOperationImpl:getRecordByIdentifier: query: "